locales/sv_SE/pcsx2_Iconized.po

```diff
@@ -425,18 +442,19 @@ msgid ""
 msgstr ""
 "Markera för att få Muspekaren att döljas då den är i ett GS-fönster;\n"
 " användbart vid brukande av musen som främsta styrmojäng för spelande.\n"
-"Som praxis gömmer sig Muspekaren automatiskt efter 2 sekunders inaktivitet."
+"Som standard gömmer sig Muspekaren automatiskt\n"
```

Comment:
```suggestion
"Som standard gömmer sig muspekaren automatiskt\n"
```

Reply:
I like "Muspekaren" over "muspekaren" since it makes it more readable.